tests/01-basic-deployment.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+import unittest
+import amulet
+
+
+class TestDeploy(unittest.TestCase):
+    """
+    Trivial deployment test for Apache Zookeeper.
+
+    This charm cannot do anything useful by itself, so integration testing
+    is done in the bundle.
+    """
+
+    def test_deploy(self):
+        self.d = amulet.Deployment(series='trusty')
+        self.d.add('apache-zookeeper', 'apache-zookeeper')
```

Comment:
The second parameter here needs to be the layer name as defined in the metadata.yamle, or else we will be testing another layer